dev/com.ibm.ws.repository.parsers/src/com/ibm/ws/repository/parsers/ParserBase.java

```diff
@@ -310,15 +310,11 @@ private void deleteDir(File dir) {
             }
             // We are either a file or the contents of the dir have been deleted, so nuke it now
             if (!f.delete()) {
-                System.out.println("Failed to delete " + f.getAbsolutePath() + " going to wait half a second and try again");
                 try {
                     // Just in case all file locks aren't quite released, try again after half a second
                     Thread.sleep(500);
                 } catch (InterruptedException ie) {
                     //swallow it
-                    if (!f.delete()) {
```

Comment:
Should it still run `f.delete()` here by itself?  Otherwise the file won't get deleted after the sleep.

Reply:
Yes, it should :( I'll open a new PR

Reply:
Fixed under #5403